config/routes.rb

```diff
@@ -14,6 +14,7 @@
     get "" => "documents#show", as: :document
     patch "" => "documents#update"
     delete "" => "documents#destroy"
+    get "/document-history" => "documents#history", as: :history_path
```

Comment:
I think `/history` would be a more appropriate path here since the full url will be `/documents/:document/history`